src/components/NavBar/NavBar.js

```diff
@@ -9,9 +10,9 @@ const NavBar = () => {
         </div>
         <div className={styles.links}>
           <ul>
-            <li><a href="/">Home</a></li>
-            <li><a href="/favorite">Favorite</a></li>
-            <li><a href="/about">About</a></li>
+            <li><NavLink className={({ isActive }) => isActive ? styles.linkActive : undefined} to="/">Home</NavLink></li>
```

Comment:
- Powtarzasz ten sam kod 3 razy, wiec to sugeruje, że warto to wynieść wyżej i np. przypisać sobie klasę do zmiennej i tylko ją przekazać. 
- na codzień raczej nie używamy "undefined" - jest to taki dziwny twór początków javascriptu, którego lepiej unikać. Zamiast tego używaj wartości "null" - null czyli po prostu nic. W tym przypadku jednak przypisujesz coś do atrybutu className, więc dobrze przypisać mu typ danych jaki oczekuje, czyli string - więc najlepiej użyć tutaj po prostu pustego stringa.
- tak ku pamięci - normalnie do className lub styles przypisujemy stringa - jednak komponent NavLink z react-routera traktuje to jak callback, stąd ten `isActive` pochodzi z "wewnętrznego" stanu tego komponentu, więc dla innych komponentów ta sztuczka z funkcją nie zadziałą (oczywiście sami możemy sobie coś takiego oprogramować)
- w kodilli często polecana jest biblioteka "clsx" do warunkowego przypisaywania stylów (ale to tylko tak na marginesie) https://www.npmjs.com/package/clsx

Reply:
Poprawione